documentation/book/assembly-metrics.adoc

```diff
@@ -0,0 +1,25 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-metrics-{context}']
+
+= Prometheus metrics
+
+{ProductName} supports Prometheus metrics.
```

Comment:
{ProductName} supports Prometheus metrics.
When metrics are enabled, they are exposed on port 9404.
It is using link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^] to convert the JMX metrics supported by Apache Kafka and Zookeeper to Prometheus metrics.

TO

{ProductName} supports Prometheus metrics using link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^] to convert the JMX metrics supported by Apache Kafka and Zookeeper to Prometheus metrics.
When metrics are enabled, they are exposed on port 9404.